tests/format/scalafmt/test

```diff
@@ -1,5 +1,18 @@
 #!/bin/bash -e
 . "$(dirname "$0")"/../../common.sh
 
+# scala format test
 bazel test :formatted
 ! bazel test :unformatted || false
+
```

Comment:
Can we add a test that:
1. shows file is misformatted
2. reformats the file
3. shows the reformatted file passing the original test

Reply:
Yes, that makes sense. Please take a look again at `/tests/format/scalafmt/test`. Since there is no runfiles directory with non-default output, the path in format template is a bit tricky.